src/controllers/index.js

```diff
@@ -0,0 +1,3 @@
+import registerUser from './users';
+
+export default registerUser;
```

Comment:
Why are you exporting `registerUser` as default? This file is supposed to contain all the controllers for the application

Reply:
> Why are you exporting `registerUser` as default? This file is supposed to contain all the controllers for the application

@eokwukwe that was what I implemented before, but because we just one method available in the user's controller file, hound keeps giving an error of `export default is better`,  so I had to change it to this format.  As we add more functions to the user's controller we will use `export { aMethod, anotherMethod }`  instead of `export default`